cluster/up.sh

```
@@ -16,20 +16,29 @@
 
 set -ex
 
-source ./cluster/cluster.sh
-cluster::install
+SCRIPTS_PATH="$(dirname "$(realpath "$0")")"
+source ${SCRIPTS_PATH}/cluster.sh
```

Shellcheck (0.11.0)

[info] Not following: ./cluster.sh was not specified as input (see shellcheck -x).

(SC1091)